### PR TITLE
Fix Errors in Tone Mapping Demo

### PR DESCRIPTION
**Pull request by Mugen87**
see #8445

### CODE REVIEW

File: examples/js/postprocessing/AdaptiveToneMappingPass.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```
Comment (WestLangley) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: What is the reason for this change to `RGBAFormat`?

Comment (Mugen87) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: Same as in #8415. Firefox does not render with `RGBFormat` and reports the well-known message:

_Error: WebGL: drawElements: Incomplete framebuffer: Status 0x8cd6: COLOR_ATTACHMENT0 has an effective format of RGB32F, which is not renderable_

Comment (WestLangley) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: Is that a Firefox issue, then?

/ping @MiiBond

**Comment by mrdoob** [comment]
Merging because it also fixes the `LinearToGamma` breakage. But would indeed be good to hear from @MiiBond.

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/postprocessing/AdaptiveToneMappingPass.js (hunk 1 above)
Comment (MiiBond) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: Changing to RGBA shouldn't affect anything so, yeah, it's fine if it gets around the FF issue. Not sure off the top of my head whether this limitation is part of the WebGL standard or just an FF thing.

**Comment by WestLangley** [comment]
@Mugen87 Maybe it would be a good idea to make an inline note in the code so users will know why RGBA is used. Just a suggestion...

    // was RGB format. changed to RGBA format. see discussion in #8415 / #8450

File: examples/js/postprocessing/AdaptiveToneMappingPass.js (hunk 1 above)
Comment (WestLangley) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: Thanks @MiiBond. If you discover an answer, please post. The code does appear to work on other browsers.

Comment (makc) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1233046) bug was fixed today, and its duplicates 1245631 mention code 0x8cd6 and 1245850 mention GL_RGBA :) that's some coarse match, could not be bothered to read details

Comment (MiiBond) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: @WestLangley . I just ran into this today in my code and needed to change my render buffer to RGBA. Apparently, in the ```WEBGL_color_buffer_float``` extension, only RGBA32F is renderable. See https://www.khronos.org/registry/webgl/extensions/WEBGL_color_buffer_float/

In OpenGL ES, there's more detail of what's supported for renderbuffers and what is only for textures: https://www.khronos.org/registry/gles/extensions/EXT/EXT_color_buffer_float.txt. Note that RGB32F is a texture-only format.

Interestingly, if you use half-float buffers, you can render to a target without alpha. See https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_half_float/

Comment (WestLangley) [inline examples/js/postprocessing/AdaptiveToneMappingPass.js]: Many thanks, @MiiBond
